src/browser/modules/Sidebar/relate-scripts.utils.js

```diff
@@ -0,0 +1,34 @@
+import remote from 'services/remote'
+import { SLASH, CYPHER_FILE_EXTENSION } from 'shared/services/export-favorites'
+import { pick } from 'lodash-es'
+import uuid from 'uuid'
+
+// @todo: Add documentation...
+
+// will be a function once projectId is available
+export const getProjectFilesQueryVars = {
+  projectId: 'project-03688c10-a811-4c0c-85d4-581e88c2183a',
+  filterValue: CYPHER_FILE_EXTENSION
+}
+
+export const removeProjectFileMutationVars = filePath => ({
+  ...pick(getProjectFilesQueryVars, ['projectId']),
+  filePath
+})
+
+export const mapRelateFavorites = async ({
+  downloadToken,
+  name,
+  directory
+}) => ({
+  id: uuid.v4(),
+  name,
+  path: directory.startsWith('.') ? SLASH : `${SLASH}${directory}`, // @todo: need to look into this
```

Comment:
couldnt get this working with `.` which is the root returned from Relate. Had to use `SLASH` for now just to get it to show, will look into later as its not a blocking issue